assets/capes.json

```diff
@@ -335,4 +335,8 @@
     "uuid": "61f845f0-2502-4459-9fb4-e46d2600281c",
     "url": "https://raw.githubusercontent.com/kami-blue/assets/assets/assets/capes/cape13.png"
   }
```

Comment:
you're missing a comma here >:(

run it through something like https://jsonlint.com lol